sjb/config/test_cases/test_branch_origin_extended_conformance_install_update_system_containers.yml

```diff
@@ -0,0 +1,249 @@
+           sudo mv "${plugin}.py" /usr/share/ansible/plugins/callback
+        done
+        sudo sed -r -i -e 's/^#?stdout_callback.*/stdout_callback = default_with_output_lists/' -e 's/^#?callback_whitelist.*/callback_whitelist = generate_junit/' /etc/ansible/ansible.cfg
+    - type: "script"
+      title: "determine the release commit for origin images and version for rpms"
+      repository: "origin"
+      script: |-
+        if [[ -n "${PULL_REFS:-}" ]]; then
+          export OPENSHIFT_ANSIBLE_TARGET_BRANCH="${PULL_REFS%%:*}"
+        fi
+        if [[ "${OPENSHIFT_ANSIBLE_TARGET_BRANCH:-}" == "release-1.5" ]]; then
+            echo "[INFO] System containers are not supported, skipping."
+            exit 0
+        fi
+
+        jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
+        git log -1 --pretty=%h >> "${jobs_repo}/ORIGIN_COMMIT"
+        git describe --abbrev=0 >> "${jobs_repo}/ORIGIN_TAG"
+        ( source hack/lib/init.sh; os::build::rpm::get_nvra_vars; echo "-${OS_RPM_VERSION}-${OS_RPM_RELEASE}" ) >> "${jobs_repo}/ORIGIN_PKG_VERSION"
+        ( source hack/lib/init.sh; os::build::rpm::get_nvra_vars; echo "${OS_RPM_VERSION}" | cut -d'.' -f2 ) >> "${jobs_repo}/ORIGIN_PKG_MINOR_VERSION"
+    - type: "script"
+      title: "install origin"
+      repository: "aos-cd-jobs"
+      script: |-
+        if [[ -n "${PULL_REFS:-}" ]]; then
+          export OPENSHIFT_ANSIBLE_TARGET_BRANCH="${PULL_REFS%%:*}"
+        fi
+        if [[ "${OPENSHIFT_ANSIBLE_TARGET_BRANCH:-}" == "release-1.5" ]]; then
+            echo "[INFO] System containers are not supported, skipping."
+            exit 0
+        fi
+
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e containerized=true      \
+                         -e deployment_type=origin  \
+                         /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml
+        # install atomic rpm
+        sudo yum install atomic -y
+        # only origin, openvswitch and node docker images are run as system containers
+        sudo atomic pull --storage ostree docker:openshift/origin:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/openvswitch:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/node:$( cat ./ORIGIN_TAG )
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e openshift_use_system_containers=true \
+                         -e openshift_docker_systemcontainer_image_registry_override=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7 \
```

Comment:
The ``*_image_registry_override`` variables have been replaced with ``*_image_override`` variables. They expect a full image such as ``brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/container-engine:tag``

``openshift_docker_systemcontainer_image_override``

Reply:
Renamed